samples/cpp/smiledetect.cpp

```diff
@@ -83,7 +75,7 @@ int main( int argc, const char** argv )
         }
         else if( argv[i][0] == '-' )
         {
-            cerr << "WARNING: Unknown option " << argv[i] << endl;
+            cerr << "WARNING: Unknown option %s" << argv[i] << endl;
```

Comment:
Why would you add the %s? Since argv[i] is already a charpointer, this is unecessary,